src/test/kotlin/no/nav/syfo/personstatus/api/v2/PersonoversiktStatusApiV2Spek.kt

```diff
@@ -312,6 +319,34 @@ object PersonoversiktStatusApiV2Spek : Spek({
                     }
                 }
 
+                it("should not return list of PersonOversiktStatus, if there is a person with a relevant active Oppfolgingstilfelle, and person is DIALOGMOTEKANDIDAT but not after start of oppfolgingstilfelle") {
```

Comment:
Kanskje gjenbruke ordet historisk i testbeskrivelsen? Feks:  [...], and person is a historic DIALOGMOTEKANDIDAT. Da trenger vi ikke "but not after start of oppfolgingstilfelle".